raster/src/main/scala/geotrellis/raster/io/geotiff/CompressedBytesArray.scala

```diff
@@ -0,0 +1,71 @@
+package geotrellis.raster.io.geotiff
+
+import geotrellis.vector.Extent
+import geotrellis.raster._
+import geotrellis.raster.io.geotiff._
+import geotrellis.raster.io.geotiff.reader._
+import geotrellis.raster.io.geotiff.tags._
+import geotrellis.raster.io.geotiff.util._
+
+import java.nio.ByteBuffer
+import monocle.syntax.apply._
+import spire.syntax.cfor._
+
+class CompressedBytesArray(compressedBytes: Array[Array[Byte]]) extends CompressedBytes {
```

Comment:
Sorry about that, I had begun finishing up the unit tests, but I felt that I was late to make a PR so I just added my code without the tests. I'll be adding the tests soon.